### DISCUSSION TITLE
SDK Version of "merged" sdks in VS Code status bar

### DISCUSSION THREAD

**Discussion by MarcoDc92** [BowlerHatLLC/vscode-as3mxml#606] [Q&A] [answered]
Hi,

I've done a "merge" of Harman "AIR SDK for Flex Developers - to be overlaid onto a Flex SDK" and Apache Flex 4.16 (AIR 32). When I set the merged directory as SDK used by the extension in VS Code, the status bar of IDE shows the same version of Apache Flex 4.16 (AIR 32).
Which file I have to edit to show also the Harman SDK version (33.1.1.821)?

p.s. this is only a question about the label in VS Code status bar; the extension, language server and debugging works fine.

Thank you

**Comment by joshtynjala** [comment, marked as answer]
The name in the status bar comes from one of these files (in your case, it's the first one):

- _flex-sdk-description.xml_ for the Flex SDK
- _royale-sdk-description.xml_ for Apache Royale
- _air-sdk-description.xml_ for the AIR SDK & Compiler
